### ISSUE TITLE
Vivecraft instantly crashes Minecraft!

### ISSUE THREAD

**Issue by TinyDerp** [ferriarnus/VivecraftForge#252]
Vivecraft on 1.20.1 causes the game to instantly crashes when booted into VR mode or when joining into multiplayer servers with VR off. I'm on a meta quest 3 if that's needed.
```
---- Minecraft Crash Report ----
// Daisy, daisy...

Time: 2024-02-18 15:01:37
Description: Unexpected error

java.lang.NullPointerException: Cannot invoke "net.minecraft.class_5944.method_34583(String, Object)" because "$$2" is null
	at net.minecraft.class_291.method_34431(class_291.java:130)
	at net.minecraft.class_291.method_34427(class_291.java:123)
	at net.minecraft.class_286.method_43438(class_286.java:33)
	at net.minecraft.class_286.method_43433(class_286.java:26)
	at org.vivecraft.client_vr.render.helpers.VREffectsHelper.renderMenuPanorama(VREffectsHelper.java:329)
	at org.vivecraft.client_vr.render.helpers.VREffectsHelper.renderGuiLayer(VREffectsHelper.java:865)
	at net.minecraft.class_757.handler$cdp000$vivecraft$mainMenu(class_757.java:6485)
	at net.minecraft.class_757.method_3192(class_757.java:897)
	at org.vivecraft.client_vr.render.helpers.VRPassHelper.renderSingleView(VRPassHelper.java:35)
	at net.minecraft.class_310.handler$ccg000$vivecraft$renderVRPasses(class_310.java:13456)
	at net.minecraft.class_310.method_1523(class_310.java:1220)
	at net.minecraft.class_310.method_1514(class_310.java:802)
	at net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:243)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:278)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_291.method_34431(class_291.java:130)
	at net.minecraft.class_291.method_34427(class_291.java:123)
	at net.minecraft.class_286.method_43438(class_286.java:33)
	at net.minecraft.class_286.method_43433(class_286.java:26)
	at org.vivecraft.client_vr.render.helpers.VREffectsHelper.renderMenuPanorama(VREffectsHelper.java:329)
	at org.vivecraft.client_vr.render.helpers.VREffectsHelper.renderGuiLayer(VREffectsHelper.java:865)
	at net.minecraft.class_757.handler$cdp000$vivecraft$mainMenu(class_757.java:6485)
	at net.minecraft.class_757.method_3192(class_757.java:897)
	at org.vivecraft.client_vr.render.helpers.VRPassHelper.renderSingleView(VRPassHelper.java:35)
Mixins in Stacktrace:
	net.minecraft.class_291:
		com.qendolin.betterclouds.mixin.VertexBufferAccessor (betterclouds.mixins.json)
		net.coderbot.iris.mixin.MixinVertexBuffer (mixins.iris.json)
	net.minecraft.class_286:
		com.qendolin.betterclouds.mixin.BufferRendererAccessor (betterclouds.mixins.json)
	net.minecraft.class_757:
		com.replaymod.core.mixin.MixinGameRenderer (mixins.core.replaymod.json)
		com.replaymod.render.mixin.GameRendererAccessor (mixins.render.replaymod.json)
		me.cortex.nvidium.mixin.minecraft.MixinGameRenderer (nvidium.mixins.json)
		net.fabricmc.fabric.mixin.screen.GameRendererMixin (fabric-screen-api-v1.mixins.json)
		com.replaymod.render.mixin.Mixin_PreserveDepthDuringHandRendering (mixins.render.replaymod.json)
		net.coderbot.iris.mixin.GameRendererAccessor (mixins.iris.json)
		com.replaymod.render.mixin.Mixin_PreserveDepthDuringGuiRendering (mixins.render.replaymod.json)
		com.replaymod.lib.de.johni0702.minecraft.gui.versions.mixin.MixinGameRenderer (mixins.jgui.json)
		com.replaymod.render.mixin.Mixin_Stereoscopic_HandRenderPass (mixins.render.replaymod.json)
		org.vivecraft.mixin.client_vr.renderer.GameRendererVRMixin (vivecraft.mixins.json)
		io.wispforest.owo.mixin.shader.GameRendererMixin (owo.mixins.json)
		org.vivecraft.fabric.mixin.FabricGameRendererVRMixin (vivecraft.fabric.mixins.json)
		com.replaymod.render.mixin.Mixin_Omnidirectional_Rotation (mixins.render.replaymod.json)
		net.fabricmc.fabric.mixin.client.rendering.shader.GameRendererMixin (fabric-rendering-v1.mixins.json)
		net.coderbot.iris.mixin.MixinGameRenderer (mixins.iris.json)
		com.replaymod.render.mixin.Mixin_Stereoscopic_Camera (mixins.render.replaymod.json)
		net.coderbot.iris.mixin.MixinModelViewBobbing (mixins.iris.json)
		me.jellysquid.mods.sodium.mixin.features.gui.hooks.console.GameRendererMixin (sodium.mixins.json)
		com.replaymod.replay.mixin.Mixin_ShowSpectatedHand_NoOF (mixins.replay.replaymod.json)
		com.replaymod.replay.mixin.MixinCamera (mixins.replay.replaymod.json)
		net.coderbot.iris.mixin.MixinGameRenderer_NightVisionCompat (mixins.iris.json)
		com.replaymod.render.mixin.Mixin_Omnidirectional_Camera (mixins.render.replaymod.json)
		com.replaymod.render.mixin.MixinEntityRenderer (mixins.render.replaymod.json)
		com.replaymod.compat.shaders.mixin.MixinShaderEntityRenderer (mixins.compat.shaders.replaymod.json)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: No
	Packs: vanilla, fabric, replaymod_lang
	Recovery: Yes
	Recovery reason: java.util.concurrent.CompletionException: java.lang.NoSuchFieldError: quads
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:791)
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)
	at net.minecraft.class_4014.method_18365(class_4014.java:69)
	at net.minecraft.class_1255.method_18859(class_1255.java:156)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:130)
	at net.minecraft.class_1255.method_5383(class_1255.java:115)
	at net.minecraft.class_310.method_1523(class_310.java:1175)
	at net.minecraft.class_310.method_1514(class_310.java:802)
	at net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:243)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:278)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
Caused by: java.lang.NoSuchFieldError: quads
	at org.vivecraft.mod_compat_vr.sodium.SodiumHelper.copyModelCuboidUV(SodiumHelper.java:58)
	at org.vivecraft.client.render.VRPlayerModel_WithArms.<init>(VRPlayerModel_WithArms.java:54)
	at org.vivecraft.client.render.VRPlayerRenderer.<init>(VRPlayerRenderer.java:29)
	at net.minecraft.class_898.handler$ccb000$vivecraft$reload(class_898.java:4106)
	at net.minecraft.class_898.method_14491(class_898.java:398)
	at net.minecraft.class_4013.method_29490(class_4013.java:15)
	at java.base/java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:787)
	... 18 more

Stacktrace:
	at net.minecraft.class_6360.method_36565(class_6360.java:49)
	at net.minecraft.class_310.method_1587(class_310.java:2413)
	at net.minecraft.class_310.method_1514(class_310.java:826)
	at net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:243)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:278)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
Mixins in Stacktrace:
	net.minecraft.class_6360:
		org.vivecraft.mixin.client.ResourceLoadStateTrackerMixin (vivecraft.mixins.json)
	net.minecraft.class_310:
		com.replaymod.lib.de.johni0702.minecraft.gui.versions.mixin.MixinMinecraft (mixins.jgui.json)
		net.fabricmc.fabric.mixin.screen.MinecraftClientMixin (fabric-screen-api-v1.mixins.json)
		net.fabricmc.fabric.mixin.event.interaction.client.MinecraftClientMixin (fabric-events-interaction-v0.client.mixins.json)
		net.coderbot.iris.mixin.MixinMinecraft_Keybinds (mixins.iris.json)
		de.maxhenkel.voicechat.mixin.MinecraftMixin (voicechat.mixins.json)
		net.fabricmc.fabric.mixin.event.lifecycle.client.MinecraftClientMixin (fabric-lifecycle-events-v1.client.mixins.json)
		me.cortex.nvidium.mixin.minecraft.MixinMinecraftClient (nvidium.mixins.json)
		squeek.appleskin.mixin.MinecraftClientMixin (appleskin.mixins.json)
		org.vivecraft.mixin.client_vr.MinecraftVRMixin (vivecraft.mixins.json)
		betteradvancements.mixin.MinecraftMixin (betteradvancements.mixins.json)
		shcm.shsupercm.fabric.citresewn.defaults.mixin.types.enchantment.MinecraftClientMixin (citresewn-defaults.mixins.json)
		net.fabricmc.fabric.mixin.networking.client.accessor.MinecraftClientAccessor (fabric-networking-api-v1.client.mixins.json)
		net.fabricmc.fabric.mixin.datagen.client.MinecraftClientMixin (fabric-data-generation-api-v1.client.mixins.json)
		io.github.thecsdev.tcdcommons.client.mixin.events.MixinMinecraftClient (tcdcommons.client.mixin.events.json)
		com.minelittlepony.common.mixin.MixinMinecraftClient (kirin.mixin.json)
		com.replaymod.lib.de.johni0702.minecraft.gui.versions.mixin.Mixin_PassEvents_HandleKeybinds (mixins.jgui.json)
		me.jellysquid.mods.sodium.mixin.core.MinecraftClientMixin (sodium.mixins.json)
		com.replaymod.render.mixin.Mixin_SuppressFramebufferResizeDuringRender (mixins.render.replaymod.json)
		net.fabricmc.fabric.mixin.registry.sync.client.MinecraftClientMixin (fabric-registry-sync-v0.client.mixins.json)
		io.wispforest.owo.mixin.MinecraftClientMixin (owo.mixins.json)
		io.wispforest.owo.mixin.ui.MinecraftClientMixin (owo.mixins.json)
		com.replaymod.core.mixin.MixinMinecraft (mixins.core.replaymod.json)
		dev.isxander.yacl3.mixin.MinecraftMixin (yacl.mixins.json)
		net.coderbot.iris.mixin.MixinMinecraft_Images (mixins.iris.json)
		net.coderbot.iris.mixin.MixinMinecraft_PipelineManagement (mixins.iris.json)
		com.replaymod.core.mixin.MinecraftAccessor (mixins.core.replaymod.json)
	net.minecraft.client.main.Main:
		org.vivecraft.mixin.client.main.MainMixin (vivecraft.mixins.json)
		link.infra.screenshotclipboard.common.mixin.AWTHackMixin (screenshotclipboard-common.mixins.json)

-- System Details --
Details:
	Minecraft Version: 1.20.1
	Minecraft Version ID: 1.20.1
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 19.0.1, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode, sharing), Oracle Corporation
	Memory: 1292907112 bytes (1233 MiB) / 1937768448 bytes (1848 MiB) up to 6442450944 bytes (6144 MiB)
	CPUs: 16
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 7 3700X 8-Core Processor             
	Identifier: AuthenticAMD Family 23 Model 113 Stepping 0
	Microarchitecture: Zen 2
	Frequency (GHz): 3.59
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 16
	Graphics card #0 name: NVIDIA GeForce RTX 3060
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x2503
	Graphics card #0 versionInfo: DriverVersion=31.0.15.5152
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.67
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 2.67
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 31692.32
	Virtual memory used (MB): 25951.21
	Swap memory total (MB): 15360.00
	Swap memory used (MB): 621.35
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms1024m -Xmx6144m
	Fabric Mods: 
		appleskin: AppleSkin 2.5.1+mc1.20
		betteradvancements: Better Advancements 0.3.2.162
		betterclouds: Better Clouds 1.3.14+1.20.1
		betterf3: BetterF3 7.0.2
		betterstats: Better Statistics Screen 3.8.1+fabric-1.20.1
			tcdcommons: TCD Commons API 3.8.1+fabric-1.20.1
		cavedust: Cave Dust 1.4.1
		citresewn: CIT Resewn 1.1.3+1.20
			citresewn-defaults: CIT Resewn: Defaults 1.1.3+1.20
		cloth-config: Cloth Config v11 11.1.118
			cloth-basic-math: cloth-basic-math 0.6.1
		continuity: Continuity 3.0.0-beta.4+1.20.1
		deathlog: DeathLog 0.3.2+1.20
		detailab: Detail Armor Bar 2.6.3+1.20.1-fabric
		eatinganimationid: Eating Animation 1.20+1.9.5
		fabric-api: Fabric API 0.92.0+1.20.1
			fabric-api-base: Fabric API Base 0.4.31+1802ada577
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.36+1802ada577
			fabric-biome-api-v1: Fabric Biome API (v1) 13.0.13+1802ada577
			fabric-block-api-v1: Fabric Block API (v1) 1.0.11+1802ada577
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.1+1802ada577
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.41+1802ada577
			fabric-client-tags-api-v1: Fabric Client Tags 1.1.2+1802ada577
			fabric-command-api-v1: Fabric Command API (v1) 1.2.34+f71b366f77
			fabric-command-api-v2: Fabric Command API (v2) 2.2.13+1802ada577
			fabric-commands-v0: Fabric Commands (v0) 0.2.51+df3654b377
			fabric-containers-v0: Fabric Containers (v0) 0.1.64+df3654b377
			fabric-content-registries-v0: Fabric Content Registries (v0) 4.0.11+1802ada577
			fabric-convention-tags-v1: Fabric Convention Tags 1.5.5+1802ada577
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.19+1802ada577
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.0.0+de0fd6d177
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 12.3.4+1802ada577
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.54+1802ada577
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.6.0+1c78457f77
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.6.2+1802ada577
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.63+df3654b377
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.40+1802ada577
			fabric-item-api-v1: Fabric Item API (v1) 2.1.28+1802ada577
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.0.12+1802ada577
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.37+1802ada577
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.35+df3654b377
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.22+1802ada577
			fabric-loot-api-v2: Fabric Loot API (v2) 1.2.1+1802ada577
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.45+9e7660c677
			fabric-message-api-v1: Fabric Message API (v1) 5.1.9+1802ada577
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.50+1802ada577
			fabric-model-loading-api-v1: Fabric Model Loading API (v1) 1.0.3+1802ada577
			fabric-models-v0: Fabric Models (v0) 0.4.2+9386d8a777
			fabric-networking-api-v1: Fabric Networking API (v1) 1.3.11+1802ada577
			fabric-networking-v0: Fabric Networking (v0) 0.3.51+df3654b377
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 11.1.3+1802ada577
			fabric-particles-v1: Fabric Particles (v1) 1.1.2+1802ada577
			fabric-recipe-api-v1: Fabric Recipe API (v1) 1.0.21+1802ada577
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 2.3.3+1802ada577
			fabric-renderer-api-v1: Fabric Renderer API (v1) 3.2.1+1802ada577
			fabric-renderer-indigo: Fabric Renderer - Indigo 1.5.1+1802ada577
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.46+df3654b377
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.37+92a0d36777
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.28+1802ada577
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.49+df3654b377
			fabric-rendering-v1: Fabric Rendering (v1) 3.0.8+1802ada577
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.3.8+1802ada577
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.11.10+1802ada577
			fabric-screen-api-v1: Fabric Screen API (v1) 2.0.8+1802ada577
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.30+1802ada577
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.13+1802ada577
			fabric-transfer-api-v1: Fabric Transfer API (v1) 3.3.4+1802ada577
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 4.3.1+1802ada577
		fabricloader: Fabric Loader 0.15.7
			mixinextras: MixinExtras 0.3.5
		indium: Indium 1.0.30+mc1.20.4
		iris: Iris 1.6.17
			io_github_douira_glsl-transformer: glsl-transformer 2.0.0-pre13
			org_anarres_jcpp: jcpp 1.4.14
			org_antlr_antlr4-runtime: antlr4-runtime 4.11.1
		java: Java HotSpot(TM) 64-Bit Server VM 19
		make_bubbles_pop: Make Bubbles Pop Mod 0.2.0-fabric
		minecraft: Minecraft 1.20.1
		mixintrace: MixinTrace 1.1.1+1.17
		modelfix: Model Gap Fix 1.14
		modmenu: Mod Menu 7.2.2
		modmenu-badges-lib: ModMenu Badges Lib 2023.6.1
		nvidium: Nvidium 0.2.6-beta
		oscimate_soulflame: Improved Flame Overlays 1.1.0
		owo: oωo 0.11.2+1.20
			blue_endless_jankson: jankson 1.2.2
		particlerain: Particle Rain 2.0.8
		presencefootsteps: Presence Footsteps 1.9.1
			kirin: Kirin UI 1.15.4+1.20
		replaymod: Replay Mod 1.20.1-2.6.14
		screenshotclipboard: Screenshot to Clipboard 1.0.9
		shulkerboxtooltip: Shulker Box Tooltip 4.0.4+1.20.1
		sodium: Sodium 0.5.8+mc1.20.1
		sound_physics_remastered: Sound Physics Remastered 1.20.1-1.3.1
		visuality: Visuality 0.7.1+1.20
		vivecraft: Vivecraft 1.20.1-1.1.5
			com_electronwill_night-config_core: core 3.6.6
			com_electronwill_night-config_toml: toml 3.6.6
			org_lwjgl_lwjgl-openvr: lwjgl-openvr 3.3.1
			org_lwjgl_lwjgl-openvr_natives-linux: lwjgl-openvr 3.3.1
			org_lwjgl_lwjgl-openvr_natives-macos: lwjgl-openvr 3.3.1
			org_lwjgl_lwjgl-openvr_natives-windows: lwjgl-openvr 3.3.1
		voicechat: Simple Voice Chat 1.20.1-2.5.4
		waterdripsound: Drip Sounds 1.19-0.3.2
		yet_another_config_lib_v3: YetAnotherConfigLib 3.2.2+1.20
			com_twelvemonkeys_common_common-image: common-image 3.10.0-SNAPSHOT
			com_twelvemonkeys_common_common-io: common-io 3.10.0-SNAPSHOT
			com_twelvemonkeys_common_common-lang: common-lang 3.10.0-SNAPSHOT
			com_twelvemonkeys_imageio_imageio-core: imageio-core 3.10.0-SNAPSHOT
			com_twelvemonkeys_imageio_imageio-metadata: imageio-metadata 3.10.0-SNAPSHOT
			com_twelvemonkeys_imageio_imageio-webp: imageio-webp 3.10.0-SNAPSHOT
			org_quiltmc_parsers_gson: gson 0.2.1
			org_quiltmc_parsers_json: json 0.2.1
	Loaded Shaderpack: (off)
	Launched Version: 1.20.1
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: NVIDIA GeForce RTX 3060/PCIe/SSE2 GL version 3.2.0 NVIDIA 551.52, NVIDIA Corporation
	Window size: 2352x2516
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'vivecraft'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: 
	Current Language: en_us
	CPU: 16x AMD Ryzen 7 3700X 8-Core Processor 
```

**Comment by fayer3** [comment]
that doesn't make sense. Are you sure it does work correctly without vivecraft? because the crash is from not being able to load the soud library.

**Comment by TinyDerp** [comment]
Might have been a wrong crash report I crashed again and got the newest error. And yes it works without Vivecraft installed.
Edit: I updated the main post to show the newest crash report,

**Comment by fayer3** [comment]
okay, that make more sense, duplicate of #247

**Comment by fayer3** [comment]
fixed with 1.1.7, now available
